sdks/java/core/src/main/java/org/apache/beam/sdk/coders/StructuredCoder.java

```diff
@@ -33,18 +32,23 @@
  * <p>To extend {@link StructuredCoder}, override the following methods as appropriate:
  *
  * <ul>
- *   <li>{@link #getComponents}: the default implementation returns {@link #getCoderArguments}.</li>
- *   <li>{@link #getEncodedElementByteSize} and
- *       {@link #isRegisterByteSizeObserverCheap}: the
+ *   <li>{@link #getComponents}: the default implementation returns {@link #getCoderArguments}.
+ *   <li>{@link #getEncodedElementByteSize} and {@link #isRegisterByteSizeObserverCheap}: the
  *       default implementation encodes values to bytes and counts the bytes, which is considered
- *       expensive.</li>
+ *       expensive.
  * </ul>
  */
 public abstract class StructuredCoder<T> extends Coder<T> {
+  /**
+   * Create a new {@link StructuredCoder}.
+   */
   protected StructuredCoder() {}
 
   /**
    * Returns the list of {@link Coder Coders} that are components of this {@link Coder}.
+   *
+   * <p>The default components will be equal to the value returned by {@link #getCoderArguments()},
```

Comment:
Since this is mentioned a couple of times, perhaps a parenthetical about what this means, like "a list associating coders with type parameters" or some such. I don't have great phrasing, so meh.